### ISSUE TITLE
cookies 

### ISSUE THREAD

**Issue by keenan35i** [Telerik-Verified-Plugins/WKWebView#206]
There is a lack in support for cookies on ios8 with this plugin, cookies work in ios9 however.... not sure what to do because i need cookies for my app to run..

**Comment by JoMas971** [comment]
Same issue for me. I can't use secured iframes because cookies are not stored/shared.

**Comment by alisiikh** [comment]
I also have the similar issue. I have an iframe inside of a WKWebView which points to some website which requires cookies and apparently WKWebView simply ignores cookies sent by the server and because of this it doesn't work. Without iframe everything is OK, but Cordova plugins will not work then. Maybe you guys have some idea for workaround of this? Both Obj-C and JavaScript workarounds will help. Thanks!

**Comment by JoMas971** [comment]
@alisiikh : are you using InAppBrowser for authentication ? (for your iFrames) Because, there is a PR waiting to be merge who consist in sharing cookies between web views. Working only for Android atm but it can be done in Obj-C :-). 
Well, the only solution that working for now is to force cookie sharing between web views. Each times you call the InAppBrowser.getCookies() method, it will share the cookie between both web-views and fix this issue... https://github.com/apache/cordova-plugin-inappbrowser/pull/122

But still a workaround.

**Comment by alisiikh** [comment]
@JoMas971 no, currently iframe is inserted directly into the body of index.html, but InAppBrowser solution sounds promising, maybe it's worth a try to use InAppBrowser for an iframe. However in this case I think a lot of stuff will need to be changed in my current implementation.

Thank you very much for your suggestion!
